ResearchKit/Common/ORKFormItemCell.h

```diff
@@ -83,4 +83,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKFormItemPickerCell : ORKFormItemTextFieldBasedCell
 @end
 
+@interface ORKFormItemScaleCell : ORKFormItemCell
++ (CGFloat)suggestedCellHeightInTableView:(nullable UITableView *)tableView;
```

Comment:
Should use auto layout like other `ORKFormItemCell`, Instead of a hard code height.